src/lib/GClient.ts

```diff
@@ -24,19 +44,90 @@ export enum AutoDeferType {
 	'UPDATE' = 3,
 }
 
+/**
+ * Options for the GClient.
+ * @extends {ClientOptions}
+ */
 export interface GClientOptions extends ClientOptions {
+	/**
+	 * Support for message commands.
+	 * @type {boolean}
+	 */
 	messageSupport?: boolean;
+	/**
+	 * Prefix for message commands
+	 * @requires {@link GClientOptions.messageSupport} to be enabled
+	 */
 	messagePrefix?:
 		| ((message: Message) => Awaitable<GClientMessagePrefix>)
 		| GClientMessagePrefix;
+	/**
+	 * Whether to send a message for an unknown command.
+	 * @type {boolean}
+	 */
 	unknownCommandMessage?: boolean;
+	/**
+	 * Array of all folders to be loaded by GCommands.
+	 *
+	 * GCommands will check all the files in each folder for these classes:
+	 * * Command
+	 * * Listener
+	 * * Component
+	 *
+	 * @type {string[]}
+	 * @example
+	 * ```typescript
+	 * new GClient({
+	 * 		dirs: [
+	 * 			join(__dirname, 'commands'),
+	 * 			join(__dirname, 'events'),
+	 * 			join(__dirname, 'components'),
+	 * 		]
+	 * })
+	 * ```
+	 */
 	dirs?: Array<string>;
+	/**
+	 * The database to be used in the project.
```

Comment:
```suggestion
	 * Database for this client
```

Reply:
It's not for client because gcommands doesn't using db